Makefile

```diff
@@ -79,7 +71,9 @@ set_pipeline: check_fly
 		--config ci/pipeline.yml \
 		--pipeline $(CONCOURSE_PIPELINE) \
 		--non-interactive \
```

Comment:
Can you add the `--check-creds` flag here too?

Reply:
Done